README.md

```diff
@@ -205,16 +205,24 @@ To do that, you can [install the `cloudflared` CLI tool](https://developers.clou
 # Note that you can also use a different port
 cloudflared tunnel --url http://localhost:3000
 ```
+Out of the logs produced by cloudflare you will notice a https URL where the domain ends with `trycloudflare.com`. This is your tunnel URL. You need to copy this URL as you will need it in the next step.
+```shell
+2022-11-11T19:57:55Z INF Requesting new quick Tunnel on trycloudflare.com...
+2022-11-11T19:57:58Z INF +--------------------------------------------------------------------------------------------+
+2022-11-11T19:57:58Z INF |  Your quick Tunnel has been created! Visit it at (it may take some time to be reachable):  |
+2022-11-11T19:57:58Z INF |  https://four-dashed-word-url.trycloudflare.com                                     |
```

Comment:
Oddly it's not always 4 four words, sometimes it's like `foo-bar--baz`.

Reply:
(so maybe call this `https://randomly-generated-hostname.trycloudflare.com`)

Reply:
Ohh out of all my 20-ish trial I had 4 words. Good spot! I'll use your suggestion!